### PR TITLE
[#268] Claim category data changes + intent threshold refactoring

### PR DESCRIPTION
**Pull request by Vynny**
[#268]

- [X] Removes entities from claim category data
- [X] Adds a few extra claim category data points for better classification
- [X] Moves intent sufficiency determination into own class
- [X] Adds basic NLP integration test 
- [X] Makes intent thresholds a bit more lenient (60% -> 40%). To be changed after working out a better solution

### CODE REVIEW

**Review by arekmano — APPROVED** [review]
looks good.

File: src/nlp_service/rasa/intent_threshold.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    Method which verifies the accuracy of the classification with a percentage difference between the intent with the largest confidence and the intent with the 2nd largest confidence
+    classify_dict: the dict holding the intents, classification %, entities
+    :returns False if percentage difference is below minimum_percent_difference
+    """
+
+    def is_sufficient(self, classify_dict):
+        if len(classify_dict['intent_ranking']) > 1:
+            percent_difference = self.intent_percent_difference(classify_dict)
+            highest_intent_confidence = classify_dict['intent']['confidence']
+            if highest_intent_confidence < self.min_confidence_threshold or percent_difference < self.min_percent_difference:
```
Comment (Samuel-Campbell) [inline src/nlp_service/rasa/intent_threshold.py:16]: Are you sure this line works? It's missing parenthesis imo

Comment (Samuel-Campbell) [reply src/nlp_service/rasa/intent_threshold.py:16]: highest_intent_confidence < self.min_confidence_threshold
self.min_confidence_threshold or percent_difference
percent_difference < self.min_percent_difference

Comment (Samuel-Campbell) [reply src/nlp_service/rasa/intent_threshold.py:16]: Nvm I tried it and it works. It doesn't work if you had a substraction or addition in the line however,

**Review by lancelafontaine — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**